py/server/tests/test_table.py

```diff
@@ -979,6 +979,20 @@ def do_something(p=None):
         self.assertEqual(df.loc[0]['Col'], 1)
         self.assertTrue(rt.columns[0].data_type == dtypes.int32)
 
+    def test_await_update(self):
+        with self.assertRaises(DHError):
+            empty_table(10).await_update()
+
+        time_t = time_table("00:00:00.001")
+        updated = time_t.await_update()
```

Comment:
Don't some of the other tests use `await_update` via `jpy`?  If so, these should be changed to use the new functionality instead of directly though `jpy`.

Reply:
No, it is a util function in the base test class.